doc/docs/using_dtm.md

```diff
@@ -42,6 +44,8 @@ To execute receiving, complete the following steps:
 1. In the Direct Test Mode app:
 
     1. Select the supported device programmed with the firmware compatible with Direct Test Mode.
+    1. Select the [**Serial port**](overview.md#select-device) from the drop-down menu.<br/>The selection should mathch the port assigned in the Device Manager.
+    1. Select the [**Baud rate**](overview.md#select-device) for the connection.
```

Comment:
most of the time users won't need to select anything and default will work (changing defaults won't work in most cases and made avaible to cover other possible edge cases. So these steps are optional.